### ISSUE TITLE
Incorrect perk number tracking for Jaws of the Lion

### ISSUE THREAD

**Issue by brother-oak** [Lurkars/gloomhavensecretariat#526]
### Describe the bug

I'm setting up an existing Jaws of the Lion campaign, transferring the data from the paper sheets into this app. When adding perks, the app works out how many perks should be allowed (one per level after level 1, and one per 3 tick marks). However, it doesn't take into account the fact that in JotL, scenario 3 gives a perk as a reward, so assuming scenario 3 has been successfully completed, it should allow an additional perk to be selected (I know you can force selection of a perk by double-clicking, but then you get a warning that you have too many selected).

I do not know if this would have been corrected if I used the app during play, but as I could not find a way of ticking anything to show that rewards have been claimed, I suspect it is a bug?

### Game Enviroment

JotL after scenario 3

### To Reproduce

Select a new Jaws of the Lion campaign
Create a new character
Go into Scenario mode and select scenarios 1, 2 and 3 as complete
Go to character sheet - all perks are greyed out although scenario 3 grants a perk as a reward

### Expected behavior

Should allow a perk to be selected after successful completion of scenario 3 (and this additional perk carries forward for the rest of the scenario - i.e. number of allowed perks = (chr level - 1) + (sets of 3 ticks) + (1 if scenario 3 complete)

### Gloomhaven Secretariat Version

v0.91.1

### Used Browser

Chrome (installed as app)

### OS

Windows 11

**Comment by Lurkars** [comment]
Look at your character sheet again! If you scroll down, you should see a box called **Extra Perks**, you can just add one and get that perk. And you're right, if you play the campaign and finish JOTL scenario 3, you'll automatically get the extra perk!
